doc/devices/dts/device_tree.rst

```diff
@@ -248,6 +248,68 @@ The following is a more precise list of required files:
     :ref:`application_dt` for more information on overlay files and the Zephyr
     build system.
 
+.. _dt-alias-chosen:
+
+``aliases`` and ``chosen`` nodes
+================================
+
+It is often useful to alias a particular node to a common name that can be
+later used in the source code in a generic way in order to avoid depending
+on the particularities of a specific board. Device Tree ``aliases`` nodes  are
+used for this purpose, by mapping certain generic, commonly used names to
+specific hardware resources:
+
+.. code-block:: yaml
+
+   aliases {
+      led0 = &led0;
+      sw0 = &button0;
+      sw1 = &button1;
+      uart-0 = &uart0;
+      uart-1 = &uart1;
+   };
+
+Certain software subsystems typically require a specific hardware resource to
+bind to in order to function properly. Some of those subsystems are typically
+used with many different boards, which makes using the Device Tree ``chosen``
+nodes very convenient. By doing so the software subsystem can rely on having
+the specific hardware peripheral assigned to it. In the following example
+we bind the shell to ``uart1`` in this board:
+
+.. code-block:: yaml
+
+   chosen {
+      zephyr,shell-uart = &uart1;
+   };
+
+The full set of Zephyr-specific ``chosen`` nodes follows:
+
+.. list-table::
+   :header-rows: 1
+
+   * - ``chosen`` node name
+     - Generated symbol
+
+   * - ``zephyr,flash``
+     - ``CONFIG_FLASH``
```

Comment:
any reason you didn't use ``:option:`CONFIG_FLASH` `` here (and on the next two items?)

Reply:
@dbkinder 
These options not exist in Kconfig so we want to keep it as non-options.